.gitignore

```diff
@@ -0,0 +1,10 @@
+/node_modules
+/dist
+#misc
+.DS_Store
+.idea
+.vscode
+.css
+.html
+.ENV
+.env
```

Comment:
Please add an extra empty line at the end of any text file you have, Github also underlines it with a red icon as you can see